jenkins/custom-ci/centos-7.9.2009/motr.groovy

```diff
@@ -78,6 +83,9 @@ pipeline {
 						else
 							./configure --with-user-mode-only
 						fi
+						if [ "${ENABLE_MOTR_DTM}" == "yes" ]; then
+							./configure --enable-dtm0
+						fi
```

Comment:
Seems with `--enable-dtm0` we are always creating packages with kernel mode. 
From example job - 
```
00:13:20.674  Wrote: /root/rpmbuild/SRPMS/cortx-motr-2.0.0-0_gitb5cd6b3c_3.10.0_1160.el7.src.rpm
00:13:47.258  Wrote: /root/rpmbuild/RPMS/x86_64/cortx-motr-2.0.0-0_gitb5cd6b3c_3.10.0_1160.el7.x86_64.rpm
00:13:47.258  Wrote: /root/rpmbuild/RPMS/x86_64/cortx-motr-devel-2.0.0-0_gitb5cd6b3c_3.10.0_1160.el7.x86_64.rpm
00:13:59.481  Wrote: /root/rpmbuild/RPMS/x86_64/cortx-motr-debuginfo-2.0.0-0_gitb5cd6b3c_3.10.0_1160.el7.x86_64.rpm
```

This option should be used in combination with `MOTR_BUILD_MODE`. Please check with Hare team on this as well.

Reply:
I have checked with Pavan Krishna, I said both options will be enable dtm0.